### PR TITLE
Update Authors

### PR DESCRIPTION
**Pull request by martinmarques**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Barman maintainers (in alphabetical order):
 
+* Andre Marchesini De Aguiar
```
Comment (barthisrael) [inline AUTHORS:3]:
```suggestion
* Andre Marchesini
```

File: doc/manual/66-about.en.md [hunk 2]
```diff
@@ -51,15 +51,20 @@ You can use GitHub's pull requests system for this purpose.
 
 In alphabetical order:
 
-* Abhijit Menon-Sen
-* Didier Michel
-* Michael Wallace
+* Andre Marchesini De Aguiar
```
Comment (barthisrael) [inline doc/manual/66-about.en.md:54]:
```suggestion
* Andre Marchesini
```

**Review by barthisrael — CHANGES_REQUESTED** [review]
Looks good to me.

Based on #998, where we initially thought of touching `AUTHORS`, @andremagui prefers to use only "2 names", so we follow the pattern of the docs.

File: AUTHORS (hunk 1 above)
Comment (gcalacoci) [inline AUTHORS:3]:
```suggestion
* Andre Marchesini
```

File: doc/manual/66-about.en.md (hunk 2 above)
Comment (gcalacoci) [inline doc/manual/66-about.en.md:54]:
```suggestion
* Andre Marchesini
```

**Review by barthisrael — APPROVED** [review]
Looks good.
We can squash and merge.

**Review by gcalacoci — APPROVED** [review]

**Pull request merged**